bakery/mgostorage/storage.go

```diff
@@ -29,43 +30,25 @@ type mgoStorage struct {
 }
 
 type storageDoc struct {
-	Location string `bson:"loc"`
+	Location string `bson:"_id"`
 	Item     string `bson:"item"`
-}
 
-func (s *mgoStorage) setUpCollection() error {
```

Comment:
Does this mean the caller is now responsible for setting up the collection index? I'm not completely sure i like this..